connectivity/netsocket/include/netsocket/DTLSSocketWrapper.h

```diff
@@ -40,6 +40,14 @@ class DTLSSocketWrapper : public TLSSocketWrapper {
      * @param control      Transport control mode. See @ref control_transport.
      */
     DTLSSocketWrapper(Socket *transport, const char *hostname = NULL, control_transport control = TRANSPORT_CONNECT_AND_CLOSE);
+
+    /** Destroy a socket wrapper.
+     *
+     *  Closes socket wrapper if the socket wrapper is still open.
```

Comment:
```suggestion
     *  Closes socket wrapper if the socket wrapper is still opened.
```

Reply:
> Add please information to the commit message (what is the issue about and how it fixes it).
> 
> Does this need also unittest addition?

I am not sure how this should be tested correctly, because a server is needed which does not respond to a DTLS handshake request.